openmrs/src/test/java/org/motechproject/openmrs/it/MRSTasksIntegrationBundleIT.java

```diff
@@ -378,32 +412,62 @@ private Task prepareCreatePatientPostActionParameterTask() {
         taskConfig.addAll(taskConfigStepSortedSet);
 
         ArrayList<TaskActionInformation> taskActions = new ArrayList();
-        taskActions.add(prepareCreatePatientActionInformation("Wallstreet 15/2", "John Smith"));
-        taskActions.add(prepareCreatePatientActionInformation("{{pa.0.uuid}}", "Jacob Lee"));
+        taskActions.add(prepareCreatePatientActionInformation("Wallstreet 15/2", "John Smith", false));
+        taskActions.add(prepareCreatePatientActionInformation("{{pa.0.uuid}}", "Jacob Lee", false));
 
         Task task = new Task("OpenMRSPatientPostActionParameterTestTask", triggerInformation, taskActions, taskConfig, true, true);
         getTaskService().save(task);
 
         return task;
     }
 
-    private TaskActionInformation prepareCreatePatientActionInformation(String adress1, String motechId) {
+    private TaskActionInformation prepareCreatePatientActionInformation(String address1, String motechId, Boolean forEncounter) {
```

Comment:
use primitive type, additionally I think it would be better to name it withPatientDataSourceBubbles